xml/System/Array.xml

```diff
@@ -2464,8 +2464,9 @@
   
  This method is an O(`n`) operation, where `n` is the product of all values in `lengths`.  
   
-   
-  
+[!NOTE]
```

Comment:
The paragraph above says "Generally, the .NET Framework class library and many programming languages do not handle nonzero lower bounds.". This is rehearsing the same fact and adds an example.

Are we happy with the duplication?